com.ibm.wala.core/src/main/java/com/ibm/wala/util/scope/JUnitEntryPoints.java

```diff
@@ -87,24 +162,20 @@
         TypeName.findOrCreateClass(targetPackageAtom, targetSimpleClassAtom);
     final Atom targetMethodAtom = Atom.findOrCreateAsciiAtom(targetMethodName);
 
-    if (DEBUG) {
-      System.err.println(("finding entrypoint " + targetMethodAtom + " in " + targetType));
-    }
+    logger.finer("finding entrypoint " + targetMethodAtom + " in " + targetType);
 
     final Set<Entrypoint> entryPts = HashSetFactory.make();
 
     for (IClass klass : cha) {
       TypeName klassType = klass.getName();
       if (klassType.equals(targetType) && isJUnitTestCase(klass)) {
-        if (DEBUG) {
-          System.err.println("found test class");
-        }
+        logger.finer("found test class");
```

Comment:
Same question regarding lack of lambda

Reply:
This one is even more obvious not to use a lambda as there's no string concatenation at all.